osbs/http.py

```diff
@@ -212,4 +212,10 @@ def json(self, check=True):
         if check and self.status_code not in (0, requests.codes.OK, requests.codes.CREATED):
             raise OsbsResponseException(text, self.status_code)
 
-        return json.loads(text)
+        try:
+            return json.loads(text)
+        except ValueError:
+            msg = '{0}Headers {1}\nContent {2}'.format('HtttpResponse has corrupt json:\n',
+                                                       self.headers, self.content)
+            logger.error(msg)
```

Comment:
How about changing `logger.error` to `logger.exception` so the originating stack trace is not lost?

Reply:
Great idea. Fixed.